app/middlewares/signUpValidator.js

```diff
@@ -0,0 +1,15 @@
+const { validationResult } = require('express-validator/check'),
```

Comment:
The idea its fine but you can improve this. Think that, if you validate 30 fields for create a user, you create 30 files for check one field. It would be great if it happened in one file ( in `signUpValidator`).

Reply:
For example, maybe create a function for check property does not empty in the body. So, you call the function with the property that you want to validate